6.0.0

```diff
@@ -0,0 +1,178 @@
+Collecting Pillow
```

Comment:
Hi - I don't think that this file should be in this merge request :-)

Reply:
Hahaha, I removed it ;)